src/Profiles/Profiles.API/Controllers/AccountController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Profiles.API.ViewModels.Account;
+using Profiles.BLL.Interrfaces;
+using Profiles.BLL.Models;
+
+namespace Profiles.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController(IAccountService accountService, IMapper mapper) : ControllerBase
+    {
+        private readonly IAccountService _accountService = accountService;
+        private readonly IMapper _mapper = mapper;
+
+        [HttpGet]
+        public async Task<IEnumerable<AccountViewModel>> GetAll(string? email, string? phoneNumber, bool? isActive, CancellationToken cancellationToken)
```

Comment:
could also be externalised to ViewModel/record

Reply:
done